### ISSUE TITLE
MVC, change __items usage to iterateItems() calls for better readability

### ISSUE THREAD

**Issue by AdSchellevis** [opnsense/core#2670]

**Comment by fichtner** [comment]
@AdSchellevis it looks finished? :)

**Comment by AdSchellevis** [comment]
@fichtner, there are some parts that still use __items, these I want to replace (both in core and plugins). The new method itself is indeed finished

**Comment by fichtner** [comment]
@AdSchellevis thanks for clarifying. leaving this open then. don't forget to enjoy your vacation!

**Comment by AdSchellevis** [comment]
`__items` isn't used anymore in core, should do the same later in plugins before closing the issue

**Comment by fichtner** [comment]
I just saw this going from patched version to another...

```
[1/1] Deinstalling opnsense-devel-19.1.b_6...
Stopping configd...done
Resetting root shell
Updating /etc/shells
Unhooking from /etc/rc
Unhooking from /etc/rc.shutdown
[1/1] Deleting files for opnsense-devel-19.1.b_6: 100%
Installing opnsense-devel-19.1.b_7...
Extracting opnsense-devel-19.1.b_7: 100%
Updating /etc/shells
Registering root shell
Hooking into /etc/rc
Hooking into /etc/rc.shutdown
Starting configd.
Keep version OPNsense\Backup\ScpSettings (0.1.0)
Keep version OPNsense\Backup\NextcloudSettings (1.0.0)
Keep version OPNsense\CaptivePortal\CaptivePortal (1.0.0)
Keep version OPNsense\Cron\Cron (1.0.1)
Keep version OPNsense\Diagnostics\Netflow (1.0.0)
Keep version OPNsense\Firewall\Alias (1.0.0)
Keep version OPNsense\IDS\IDS (1.0.1)
Keep version OPNsense\Monit\Monit (1.0.4)
Keep version OPNsense\Proxy\Proxy (1.0.1)

Fatal error: Uncaught Error: Call to undefined method OPNsense\Routes\Route::iterateItems() in /usr/local/opnsense/mvc/app/models/OPNsense/Base/BaseModelMigration.php on line 46

Error: Call to undefined method OPNsense\Routes\Route::iterateItems() in /usr/local/opnsense/mvc/app/models/OPNsense/Base/BaseModelMigration.php on line 46

Call Stack:
    0.0007     361136   1. {main}() /usr/local/opnsense/mvc/script/run_migrations.php:0
    1.5694    2133192   2. OPNsense\Routes\Route->runMigrations() /usr/local/opnsense/mvc/script/run_migrations.php:56
    1.5697    2134104   3. OPNsense\Base\BaseModelMigration->run() /usr/local/opnsense/mvc/app/models/OPNsense/Base/BaseModel.php:618
    1.5697    2134104   4. OPNsense\Base\BaseModelMigration->checkDefaults() /usr/local/opnsense/mvc/app/models/OPNsense/Base/BaseModelMigration.php:61

Writing firmware setting...done.
Configuring login behaviour...done.
Configuring system logging...done.
Message from opnsense-devel-19.1.b_7:
```

**Comment by fichtner** [comment]
(merged to stable/18.7)
